### DISCUSSION TITLE
Katex font size is way too BIG

### DISCUSSION THREAD

**Discussion by dibbla** [adityatelange/hugo-PaperMod#959] [Q&A]
Hi guys, I'm now using Katex for my formulations and I followed the [instrcutions](https://github.com/bee-san/hugo-PaperMod/commit/c5c9ffc3006f5fb499b2a8c66b94bdd3800433eb) but the font size is way too large. In addition, the formulation cannot start a new line.
I've tried to use `.katex{font-size = 1.1em}` sort of thing in every .CSS file I have in the theme dir, invain.

Any idea about the font size?

**Comment by a6o** [comment]
Probably not the best method, but I did
```
.katex {
    font-size: 1.1em !important; 
}
```
in `assets/css/extended/custom.css` and it worked.

**Reply by dibbla** [reply]
hi, thanks ❤️ 

Seems like I am putting things in a wrong place.

One more method, may not seem to be in good style, is to directly modity the style in `math.html`
